https://bugzilla.mozilla.org/show_bug.cgi?id=1353765

This is a supposed fix from bug report, not yet integrated in firefox

# HG changeset patch
# User Tom Ritter <tom@mozilla.com>
# Date 1491237602 18000
#      Mon Apr 03 11:40:02 2017 -0500
# Node ID 6acbb284e1bdb2294127e34cc1cfdb77d8a9a065
# Parent  9f1a26953dedbee4e3796ee12135b0a540028cee
Add FileSystemDirectoryEntry.h include

diff --git a/dom/filesystem/compat/FileSystemDirectoryReader.h b/dom/filesystem/compat/FileSystemDirectoryReader.h
--- a/dom/filesystem/compat/FileSystemDirectoryReader.h
+++ b/dom/filesystem/compat/FileSystemDirectoryReader.h
@@ -5,16 +5,17 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */

 #ifndef mozilla_dom_FileSystemDirectoryReader_h
 #define mozilla_dom_FileSystemDirectoryReader_h

 #include "mozilla/Attributes.h"
 #include "mozilla/ErrorResult.h"
 #include "mozilla/dom/BindingDeclarations.h"
+#include "mozilla/dom/FileSystemDirectoryEntry.h"
 #include "nsCycleCollectionParticipant.h"
 #include "nsWrapperCache.h"

 namespace mozilla {
 namespace dom {

 class Directory;
 class FileSystem;
